scripts/workers/submission_worker.py

```diff
@@ -102,7 +102,7 @@ def download_and_extract_file(url, download_location):
     try:
         response = requests.get(url)
     except Exception as e:
-        print 'Failed to fetch file from {}, error {}'.format(url, e)
+        print('Failed to fetch file from {}, error {}'.format(url, e))
```

Comment:
We haven't added support for Python 3.x as of now. So, can you please remove these changes related to Python 3.x